### ISSUE TITLE
could not find snapshot artifact

### ISSUE THREAD

**Issue by magomimmo** [ato/clojars-web#272]
I'm trying to deploy a new snapshot release of a lein-template to clojars. 

```bash
lein deploy clojars
...
Wrote /Users/mimmo/Developer/cljs-start/pom.xml
Created /Users/mimmo/Developer/cljs-start/target/lein-template-0.1.4-SNAPSHOT.jar
Retrieving cljs-start/lein-template/0.1.4-SNAPSHOT/maven-metadata.xml (1k)
    from https://clojars.org/repo/
Sending cljs-start/lein-template/0.1.4-SNAPSHOT/lein-template-0.1.4-20141110.162240-10.pom (3k)
    to https://clojars.org/repo/
Sending cljs-start/lein-template/0.1.4-SNAPSHOT/lein-template-0.1.4-20141110.162240-10.jar (22k)
    to https://clojars.org/repo/
Retrieving cljs-start/lein-template/maven-metadata.xml (1k)
    from https://clojars.org/repo/
Sending cljs-start/lein-template/0.1.4-SNAPSHOT/maven-metadata.xml (1k)
    to https://clojars.org/repo/
Sending cljs-start/lein-template/maven-metadata.xml (1k)
    to https://clojars.org/repo/
```

As you can see everything seems to work. Then if I try to create a new project  based on the above snapshot I receive the following error: 

```bash
lein new cljs-start wonderful-lib --snapshot
Retrieving cljs-start/lein-template/0.1.4-SNAPSHOT/lein-template-0.1.4-20141110.162240-10.pom from clojars
Could not find artifact cljs-start:lein-template:jar:0.1.4-SNAPSHOT
This could be due to a typo in :dependencies or network issues.
If you are behind a proxy, try setting the 'http_proxy' environment variable.
Could not find template cljs-start on the classpath.
```

The problem is that I'm not behind a proxy and the dependencies are ok. Finally, if I install the snapshot on the local repository (i.e. `lein install`), everything works as expected. 

where I'm wrong?

My best

mimmo

**Comment by tobias** [comment]
It looks like you've now released 0.1.4. Is this still an issue for you?

**Comment by tobias** [comment]
Closing. Feel free to reopen if this is still an issue.
